frontend/workflows/projectSelector/src/dash.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from "react";
-import { Box } from "@material-ui/core";
+import { Table, TableRow, Typography } from "@clutch-sh/core";
```

Comment:
fyi nothing in this file will be merged. this is just for PR reviewers in case they want to locally test